addons/l10n_ar_withholding/__init__.py

```diff
@@ -12,7 +12,7 @@
 def _l10n_ar_withholding_post_init(env):
     """ Existing companies that have the Argentinean Chart of Accounts set """
     template_codes = ['ar_ri', 'ar_ex', 'ar_base']
-    ar_companies = env['res.company'].search([('chart_template', 'in', template_codes)])
+    ar_companies = env['res.company'].search([('chart_template', 'in', template_codes)], order="parent_id DESC")
```

Comment:
Wouldn't a sorting on the `parent_path` make more sense?
Not sure the hierarchy will always work nicely otherwise

Reply:
no, the main company does not necessarily have the lowest id, and the whole hierarchy does not matter much as long as we load the most parent first (null parent id). This way any missing account or tax (or anything) will be created for the root company first and we won't create duplicates

Reply:
> the main company does not necessarily have the lowest id

But the id of the company is only at the end of the path.

If you have for some reason:
| id | parent_id | parent_path |
| - | - | - |
| 1 | 3 | 2/3/1/ |
| 2 |   | 2/ |
| 3 | 1 | 1/3/ |

Your version will do 2 first, then either 3 or 1 depending on the `_order`, but not on the hierarchy. But we want it done in order: 2 then 3 then 1.

Reply:
okay, updated